README.md

```diff
@@ -29,27 +30,44 @@ They are Java Spring Boot applications, released as separate docker images.
     ├── gpc-api-facade              # GPC API Facade
     └── mhs-adaptor-mock            # Dockerfile and required files for mock of MHS Adaptor
 
-## How to start local environment
-1. Export following env variables:
+## Local development
+### How to start local environment
+1. Go to `docker` directory
+2. Set following env variables inside `vars.sh` file:
     - PS_DB_URL: Database URL required to run migrations (for local environment set 'jdbc:postgresql://localhost:5436/patient_switching')
     - POSTGRES_PASSWORD: Password to be set for default postgres user. 
       This user is used to run the init script on the database. It is required during database creation.
     - PS_DB_OWNER_NAME: Username of user used to run migrations.
     - PS_DB_OWNER_PASSWORD: Password to be set for the user used to run migrations.
     - GPC_USER_DB_PASSWORD: Password for the user connecting to the database in the GPC API Facade module.
     - GP2GP_USER_DB_PASSWORD: Password for the user connecting to the database in the GP2GP Translator module.
```

Comment:
Should there be env variables relevant to the queue implemented in this vars file?